### PR TITLE
Add public access for problems

### PR DESCRIPTION
**Pull request by danialmalik**
## Story link:
https://edlyio.atlassian.net/browse/EDE-434

## Description:
Add public access for Drag n Drop problems.

### CODE REVIEW

**Review by imhassantariq — APPROVED** [review]

File: drag_and_drop_v2/drag_and_drop_v2.py [hunk 1]
```diff
@@ -340,14 +352,19 @@ def items_without_answers():
                     item['expandedImageURL'] = ''
             return items
 
+        # Since we cannot track attempts for anonymous users, we will not keep
+        # track of attempt count and neither will show this. Anonymous user
```
Comment (HamzaIbnFarooq) [inline drag_and_drop_v2/drag_and_drop_v2.py]:
```suggestion
        # track of attempt count and neither will show this. Anonymous users
```

File: drag_and_drop_v2/drag_and_drop_v2.py [hunk 2]
```diff
@@ -217,6 +217,12 @@ def max_score(self):  # pylint: disable=no-self-use
         """
         return 1
 
+    def is_user_anonymous(self):
```
Comment (HamzaIbnFarooq) [inline drag_and_drop_v2/drag_and_drop_v2.py]: nit:
```suggestion
    def is_anonymous_user(self):
```

**Review by HamzaIbnFarooq — APPROVED** [review]

File: drag_and_drop_v2/drag_and_drop_v2.py (hunk 1 above)
Comment (danialmalik) [reply drag_and_drop_v2/drag_and_drop_v2.py]: done

File: drag_and_drop_v2/drag_and_drop_v2.py (hunk 2 above)
Comment (danialmalik) [reply drag_and_drop_v2/drag_and_drop_v2.py]: done

**Pull request merged**